src/tensorlake/documentai/parse.py

```diff
@@ -67,11 +67,14 @@ class ExtractionOptions(BaseModel):
     Options for structured data extraction.
     """
 
-    schema: Union[Type[BaseModel], Json]
+    json_schema: Union[Type[BaseModel], Json] = Field(..., alias="schema")
     prompt: Optional[str] = None
-    provider: ModelProvider = ModelProvider.TENSORLAKE
+    provider: str = "TENSORLAKE"  # Assuming you replace this with your enum
     skip_ocr: bool = False
 
+    class Config:
+        allow_population_by_field_name = True  # Enables usage of 'json_schema=' as well
```

Comment:
```suggestion
        allow_population_by_field_name = True  # Enables usage of 'schema=' as well
```
I think you meant that ☝️ ?

Reply:
Yes thank you!